### DISCUSSION TITLE
Enclosure assembly | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1709] [Q&A]
# Enclosure assembly | Seeed Studio Wiki

How to assemble the enclosure

https://wiki.seeedstudio.com/respeaker_enclosure/

**Comment by mechatrocity** [comment]
This page should probably link to https://files.seeedstudio.com/products/110991984/Assembly%20instructions%20for%20Acrylic%20Enclosure.pdf

**Reply by MatthewJeffson** [reply]
Indeed it is, there might be something wrong when we launch wiki.
Thanks for the pointing!
